src/ui/mesh/qgsmeshtransformcoordinatesdialogbase.ui

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>QgsMeshTransformCoordinatesDialogBase</class>
+ <widget class="QDialog" name="QgsMeshTransformCoordinatesDialogBase">
```

Comment:
inherit from QgsDockWidget similarly to e.g. https://github.com/qgis/QGIS/blob/master/src/ui/qgsadvanceddigitizingdockwidgetbase.ui ?